.eslintignore

```diff
@@ -1,6 +1,8 @@
 build
 dist
 node_modules
-test
 vendor
 wordpress
+*.spec.js
+*.spec.js.snap
+!*/helpers.js
```

Comment:
Because we don't know how our `test/` directories could evolve in the future we want to ignore them completely. If we want to run linting on the test files, use the negation syntax like you did for the helpers.js file to pick enable linting on `*.spec.js` and `*.cypress.js` files.